src/votekit/cleaning.py

```diff
@@ -173,3 +172,27 @@ def remove_from_ballots(ballot: Ballot, non_cands: list[str]) -> Ballot:
     new_ballots = [merge_ballots(b) for b in grouped_ballots]
 
     return PreferenceProfile(ballots=new_ballots)
+
+
+def undervote_profile(pp: PreferenceProfile) -> PreferenceProfile:
+    """
+    takes a ballot and truncates its rankings in the case.
+    Args:
+        ballot (Ballot): a ballot with empty ranks in a particular ranking list
+
+    Returns:
+        Ballot: a ballot without emptinesss
+    """
+
+    def undervote(ballot: Ballot) -> Ballot:
+        rank_list = ballot.ranking
+        cleaned_rank_list = [rank for rank in rank_list if " " not in rank]
```

Comment:
the parser defaults empty cells (aka undervotes) as `None` so lets replace `'' ''` with `None`

Reply:
got it